models.py

```diff
@@ -0,0 +1,21 @@
+from pydantic import BaseModel
+
+class Coffee(BaseModel):
+    id: str = None
+    name: str
+    origin:str
+    steps: list[str]
+    body: str

```

Comment:
what is body? just curious

Reply:
It is feeling of heaviness, strenght, fatness what we can feel in our mouth and tonque

Reply:
https://www.youtube.com/watch?v=lDJ5XD5nhyg&ab_channel=AvatarAang132